### PR TITLE
WIP Feature/out of range#707

### PR DESCRIPTION
**Pull request by 121watts**
- [ ] CHANGELOG.md updated
  - [x] Rebased/mergable
  - [x] Tests pass
  - [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

Connect #707 
Connect #711 

### The problem
Nothing in the GUI or backend for 'out of range' functionality in Chronograf.

### The Solution
Add it!

### Note
Should probably hold off merging this until the backend work is done for this.

Picture's worth some words:

### CODE REVIEW

File: chronograf.go [hunk 1]
```diff
@@ -143,9 +143,9 @@ type TriggerValues struct {
 	Period        string `json:"period,omitempty"`         // Period length of time before deadman is alerted
 	Shift         string `json:"shift,omitempty"`          // Shift is the amount of time to look into the past for the alert to compare to the present
 	Operator      string `json:"operator,omitempty"`       // Operator for alert comparison
-	RangeOperator string `json:"range_operator,omitempty"` // RangeOperator is an optional operator for range comparisons
+	RangeOperator string `json:"rangeOperator,omitempty"` // RangeOperator is an optional operator for range comparisons
```
Comment (goller) [inline chronograf.go]: Please run `go fmt ./...`

File: ui/package.json [hunk 2]
```diff
@@ -23,6 +26,7 @@
     }
   },
   "devDependencies": {
+    "@kadira/storybook": "^2.21.0",
```
Comment (goller) [inline ui/package.json]: Would you update the LICENSE_OF_DEPENDENCIES.md?

File: ui/stories/stubs/kapacitor.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+const kapacitor = () => {
+  return ({
+    "id": "\"1\"",
+    "name": "kapa",
+    "url": "http://chronograf.influxcloud.net:9092",
+    "username": "hunter",
+    "password": "TMnN5JyM35H9m6Hc",
```
Comment (goller) [inline ui/stories/stubs/kapacitor.js]: Is this a real password? Probably shouldn't be checked in.

File: chronograf.go (hunk 1 above)
Comment (goller) [inline chronograf.go]: Oh also, would you update the documentation in server/swagger.json with the new names?

File: ui/stories/stubs/kapacitor.js (hunk 3 above)
Comment (cryptoquick) [inline ui/stories/stubs/kapacitor.js]: Haha, wow... My bad. That was absolutely my password.
I'll get that changed right away. I'm not sure how to change it though?
I guess I wasn't expecting that to be kept plaintext in memory on the frontend? Is there any reason for that?

File: ui/package.json (hunk 2 above)
Comment (cryptoquick) [inline ui/package.json]: Is this auto-generated?

File: chronograf.go (hunk 1 above)
Comment (cryptoquick) [inline chronograf.go]: Good catch about the go_fmt. Is the swagger.json hand-edited? I don't recall how to view and work with it.

File: ui/package.json (hunk 2 above)
Comment (goller) [inline ui/package.json]: Thankfully, it looks like I remembered to check in my script to do it: `etc/licenses.sh` ... although it looks like the Go licenses are not generated correctly from that script.  Anyway, the javascript licenses use this: 
```sh
nlf -c |awk -F, '{printf "%s %s \[%s\]\(%s\)\n", $1, $2, $5, $4}'
```

File: chronograf.go [hunk 4]
```diff
@@ -140,12 +140,13 @@ type Ticker interface {
...
-	Period     string `json:"period,omitempty"`     // Period length of time before deadman is alerted
-	Shift      string `json:"shift,omitempty"`      // Shift is the amount of time to look into the past for the alert to compare to the present
-	Operator   string `json:"operator,omitempty"`   // Operator for alert comparison
-	Value      string `json:"value,omitempty"`      // Value is the boundary value when alert goes critical
-	RangeValue string `json:"rangeValue,omitempty"` // RangeValue is an optional value for range comparisons
+	Change        string `json:"change,omitempty"`        // Change specifies if the change is a percent or absolute
+	Period        string `json:"period,omitempty"`        // Period length of time before deadman is alerted
+	Shift         string `json:"shift,omitempty"`         // Shift is the amount of time to look into the past for the alert to compare to the present
+	Operator      string `json:"operator,omitempty"`      // Operator for alert comparison
+	RangeOperator string `json:"rangeOperator,omitempty"` // RangeOperator is an optional operator for range comparisons
```
Comment (goller) [inline chronograf.go]: RangeOperator is no longer needed (merge issue)

**Pull request merged**